tests/functional/commands/conftest.py

```diff
@@ -49,30 +50,18 @@ def clean_db(database, db_instance):
     database.resources.drop()
 
 
-@pytest.fixture()
-def db_instance(null_db_instances):
-    """Create and save a singleton database instance."""
-    try:
-        db = Database(
-            of_type="MongoDB", name="orion_test", username="user", password="pass"
-        )
-    except ValueError:
-        db = Database()
-
-    return db
-
-
 @pytest.fixture
-def only_experiments_db(clean_db, database, exp_config):
+def only_experiments_db(storage, exp_config):
     """Clean the database and insert only experiments."""
-    database.experiments.insert_many(exp_config[0])
+    for exp in exp_config[0]:
+        storage.create_experiment(exp)
 
 
 def ensure_deterministic_id(name, db_instance, version=1, update=None):
```

Comment:
We could use `storage.create_experiment` instead. I think we may be able to drop the fixture `pdatabase` (and `database`) altogether.